src/encoding/encodeTransaction.ts

```diff
@@ -0,0 +1,5 @@
+import { Tx } from '../proto/cosmos/tx/v1beta1/tx';
```

Comment:
@stephensprinkle curious if you think we should just re-export this whole thing rather than wrap it

Reply:
Scoping to just what we’ve tested + need would be my personal preference here.